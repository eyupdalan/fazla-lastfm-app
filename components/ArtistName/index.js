import Link from "next/link";
import styles from "../../styles/ArtistName.module.css";

export default function ArtistName({mbid, name}) {
    return (
        <div className={styles.artistName}>
            <Link href={`/artist/${mbid}`}>
                {name}
            </Link>
        </div>
    )
}