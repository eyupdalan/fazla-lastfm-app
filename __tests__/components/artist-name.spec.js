import { render } from '@testing-library/react'
import ArtistName from '../../components/ArtistName';

it('renders Artist Name component', () => {
    const { container } = render(<ArtistName name={"test"} mbid={"deneme"} />)
    expect(container).toMatchSnapshot()
})
