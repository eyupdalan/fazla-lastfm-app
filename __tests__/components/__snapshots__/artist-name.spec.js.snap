// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Artist Name component 1`] = `
<div>
  <div
    class="artistName"
  >
    <a
      href="/artist/deneme"
    >
      test
    </a>
  </div>
</div>
`;
